WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -7532,6 +7537,20 @@
 			shellPath = /bin/sh;
 			shellScript = "/usr/bin/xcrun simctl uninstall booted org.wordpress";
 		};
+		9879533A2135D77500743763 /* Zendesk Strip Frameworks */ = {
```

Comment:
Ah, yes, the ZD SDK appears to vend a "fat" binary that includes multiple slices:
```
Tourmalet:4.1 baranski$ xcrun -sdk iphoneos lipo -info ZendeskSDK.framework/ZendeskSDK 
Architectures in the fat file: ZendeskSDK.framework/ZendeskSDK are: i386 x86_64 armv7 arm64 
Tourmalet:4.1 baranski$ xcrun -sdk iphoneos lipo -info ZendeskCoreSDK.framework/ZendeskCoreSDK 
Architectures in the fat file: ZendeskCoreSDK.framework/ZendeskCoreSDK are: i386 x86_64 armv7 arm64 
```
So this will strip out the Simulator slices prior to submission. It might be worth sharing with @loremattei as we prepare the first release with this version in it.

Reply:
Hey! Thank for the heads up @stevebaranski!!